aurora/schemas/cycling.py

```diff
@@ -100,7 +101,8 @@ def __getitem__(self, item):
 
 
 ################################################################################
-class DummyRandom(CyclingTechnique, extra=Extra.forbid):
+class DummyRandom(CyclingTechnique,
+                  extra=Extra.forbid):  # type: ignore[call-arg]
```

Comment:
Do we know what this problem is? There is quite a lot of these in this file.

Reply:
Additional arguments provided during **kwargs to class initialization (here, extra=...) seems to not be supported by mypy

Reply:
I don't yet understand why Loris added these `extra` parameters

Reply:
in this case, it was just a precaution, to make sure that a user does not add extra parameters not allowed by this data schema (normally pydantic allows them).
In general, we always wanted to make sure that these schemas were compatible with the ones read by tomato (see e.g. https://github.com/dgbowl/dgbowl-schemas/tree/master/src/dgbowl_schemas/tomato/payload_0_2 )

Reply:
Interesting. Okay, thanks. I'll read more into it as soon as I have time.

Reply:
The `extra=` are a problem I ran into when trying to get `mypy` working with `yadg`, too. I think further down the line, we still need to align the two schemas - as I mentioned before, I plan to do a rewrite of `tomato` sometime in Q3/Q4 of this year. If you get the schema here running/compatible with `mypy`, we might as well base the payload schema off of that.

Reply:
One solution is to avoid the `extra` parameter in the class signature and instead define it in a nested `Config` class. We're already doing this here in `cycling.py`, making the `extra` parameter in the signature redundant. Please correct me if I'm wrong. I plan soon to combine this in a PR resolving all `mypy` bypasses.